.github/workflows/update-docs-in-website.yml

```
@@ -0,0 +1,52 @@
+name: Update latest CLI documentation in the website
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'docs/*.md'
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest CLI documentation
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v2
+        with:
+          path: cli
+      - name: Checkout Another repository
+        uses: actions/checkout@v2
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-cli-docs-${{ github.sha }}
+      - name: Copy cli folder from Current Repo to Another
+        working-directory: ./website
+        run: |
+          mkdir -p ./pages/docs/tools/cli
+          rm ../cli/docs/README.md
+          rm -r ../cli/docs/jsdoc2md-handlebars
+          printf "%s\ntitle: cli\nweight: 3\n%s" "---" "---"> ../cli/docs/_section.md
```

```suggestion
          printf "%s\ntitle: CLI\nweight: 30\n%s" "---" "---"> ../cli/docs/_section.md
```

let's increase weight so CLI is next on the list after generator, but if needed, there is a lot of other stuff that can  be placed between these 2 in navigaton